### PR TITLE
Add historgam feature and screenshots

### PR DESCRIPTION
**Pull request by pwalczysko**
See bullet point 2 in https://github.com/ome/omero-guides/issues/2#issuecomment-648819970

Adding histogram screenshot and descript;ion to iviewer guide.

cc @jburel @will-moore

### CODE REVIEW

File: docs/iviewer.rst [hunk 1]
```diff
@@ -69,6 +71,12 @@ default viewer for OMERO.web.
 
    .. image:: images/Iviewer2.png
 
+#. Still staying in the "Settings" tab in the right-hand side of OMERO.iviewer, click on the checkbox next to the "Histogram" word |image0| near the top.
+
+#. A histogram of the first channel which is on will appear. Note that when you move the rendering settings slider beneath the histogram, the histogram reverst to showing the channel you just manipulated.
```
Comment (will-moore) [inline docs/iviewer.rst]: reverst -> reverts

**Comment by will-moore** [comment]
Looks good. Just a typo. Haven't checked layout - it is staged anywhere or I need to build locally?

**Comment by pwalczysko** [comment]
> it is staged anywhere or I need to build locally?

Locally I am afraid. Go to the ``docs`` subfolder and run ``make clean html``

File: docs/iviewer.rst (hunk 1 above)
Comment (pwalczysko) [reply docs/iviewer.rst]: @will-moore Fixed in https://github.com/ome/omero-guide-iviewer/pull/2/commits/c3a35413b99e68c242e47e58a64965841046891a

**Comment by will-moore** [comment]
Sorry, I think I was distracted by the typo, but "updates" is probably better than "reverts" since you may not be switching back to a channel you previously viewer.
Also "rendering settings slider**s**" (since there are multiple sliders).

Minor point - I wonder if you could reduce the size of the histogram screenshot (2nd image) a little. It just feels a little too "in your face". I see that the images are set to the same size in the doc (although they are very different sizes).

Thanks.

**Comment by pwalczysko** [comment]
@will-moore Made the screenshot just a little bit smaller. The reason you see the same numbers in the block below the text for all the images is that these are probably remnants from the original conversion via pandoc. These numbers are not taken into account for most of the images, because the images are not cited in the text with ``|imagex|`` in most of cases. But the ``|image0b|`` is cited properly, and thus the numbers below the text do apply for it.

**Comment by will-moore** [comment]
Thanks. Good to merge 👍

**Comment by pwalczysko** [comment]
Merging on the strenght of https://github.com/ome/omero-guide-iviewer/pull/2#issuecomment-649502565 - this will give us also a headstart on further work on the iviewer splitting https://openmicroscopy.slack.com/archives/C1PJHN2D7/p1593077207176100

cc @jburel

**Pull request merged**